File: src/main/java/vee/vaaw/Vaaw.java
package vee.vaaw;

import vee.vaaw.util.EffectHelper;
import vee.vaaw.util.ItemHelper;
import net.fabricmc.api.ModInitializer;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class Vaaw implements ModInitializer {

    public static final String MOD_ID = "vaaw";
    public static final Logger LOGGER = LoggerFactory.getLogger(MOD_ID);

    @Override
    public void onInitialize() {
        LOGGER.info("Initialising Void's Artifacts and Weapons.");
        ItemHelper.initializeItems();
        ItemHelper.initializeItemGroups();
        EffectHelper.INSTANCE.intialiseEffects();
    }
}

File: src/main/kotlin/vee/vaaw/Vaaw.kt
package vee.vaaw

import net.fabricmc.api.ModInitializer
import org.slf4j.Logger
import org.slf4j.LoggerFactory
import vee.vaaw.util.EffectHelper
import vee.vaaw.util.ItemHelper
import software.bernie.geckolib.GeckoLib
import vee.vaaw.effect.PositiveEffects
import vee.vaaw.effect.NegativeEffects
import vee.vaaw.util.PacketHelper

object Vaaw : ModInitializer {

    const val MOD_ID = "vaaw"
    val logger: Logger = LoggerFactory.getLogger(MOD_ID)

    override fun onInitialize() {
        logger.info("VAAW is now initializing.")

        GeckoLib.initialize()

        ItemHelper.initializeItems()
        ItemHelper.initializeItemGroups()
        EffectHelper.initializeEffects()
        PacketHelper.initializePackets()
    }

}